chkcryptoboot_hook

```diff
@@ -19,9 +20,35 @@ check_mbr ()
     fi
 }
 
+check_core ()
+{
+    if [ -n "$BOOTDISK" -a -s /etc/chkcryptoboot/grub-core.sha512 ]; then
+	resolve_device "$BOOTDISK"
```

Comment:
Again, same resolve_device for the BOOTDISK

Reply:
Same, to resolve #10

Reply:
No need to resolve BOOTDISK, as I have explained above.